images/Dockerfile.ubuntu

```diff
@@ -41,50 +42,51 @@ FROM ${FLAVOR}-${FLAVOR_RELEASE}-upstream AS common
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update \
-  && apt-get install -y --no-install-recommends \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
     gnupg \
     software-properties-common \
   && add-apt-repository ppa:oibaf/test \
   && apt-get update \
-  && apt-get install -y --no-install-recommends \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
     conntrack \
     console-data \
     cloud-guest-utils \
-    coreutils \
+    # coreutils \
     cryptsetup \
     curl \
     debianutils \
     dosfstools \
-    dracut \
-    dracut-network \
     e2fsprogs \
     fdisk \
     file \
     gawk \
     gdisk \
-    grub2-common \
     haveged \
     iproute2 \
     iptables \
     iputils-ping \
     jq \
-    kbd \
+    # added in https://github.com/kairos-io/kairos/commit/d413f0ec9c420a4847105e8b531e948cf4ce3176
+    # kbd \
     krb5-locales \
     less \
-    lldpd \
+    # can I remove this? it brings perl
+    # lldpd \
     lvm2 \
-    nano \
+    # there's pico
+    # nano \
     nbd-client \
-    neovim \
+    # vim is even bigger, so maybe just use pico
+    # neovim \ 
     nohang \
     open-iscsi \
     open-vm-tools \
     openssh-server \
     parted \
     rsync \
-    shim-signed \
```

Comment:
I think we need the signed shim modules everywhere to support secureboot!

Reply:
shouldn't be dumped from the LiveCD during the installation phase? 

Do we really need those in the OS runtime?

Reply:
shim-signed, has as dependency GRUB, do we also need it for UKI? and if so is there a way to get only what we need and not GRUB?

Reply:
what is the livecd but the os packed into a squashfs? We need to bundle those in the os otherwise no secureboot in non-uki installs

Reply:
yeah it's not being removed on non-uki, but do we need it for UKI?

Reply:
No, for uki is not needed, all artifacts are provided by osbuilder